### ISSUE TITLE
Usable for NixOS build farm?

### ISSUE THREAD

**Issue by Kreyren** [sipeed/LicheePi4A#19]
In the promotion it claims that the cluster is capable of parallel computing, can you elaborate if it's able to use all 28 Threads (7x module in a cluster) to do compilation?

For context i am looking for a hardware to use for the build farm in NixOS to make cached riscv packaged

Clarification: I am not in charge of the decision for the hardware infrastructure i am asking as a community member

**Comment by Zepan** [comment]
Yes, we have tested on linux kernel compile, single board use 64min, cluster use 15min

**Comment by Kreyren** [comment]
> Yes, we have tested on linux kernel compile, single board use 64min, cluster use 15min -- @Zepan (https://github.com/sipeed/LicheePi4A/issues/19#issuecomment-1713289327)

Was this done or could this be done without distcc? (nixos builds packages in a clean sandboxed environment for reproducibility so we can't do distcc)

**Comment by MaxHearnden** [comment]
While it isn't likely that a single package can compiled in a distributed manner, you could setup the modules as remote builders so you can compile multiple packages quickly at the same time. see https://nixos.org/manual/nix/stable/advanced-topics/distributed-builds for more information.

**Comment by 0x754C** [comment]
I use lm4a for guix offload, it works well, but guix haven't riscv64 bootstrap.....
